tests/unit/mocks/exceptions_mock.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2023, Denys Denysyev (ddenysyev@purestorage.com)
+# GNU General Public License v3.0+ (see COPYING.GPLv3 or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import fusion as purefusion
+
+from http import HTTPStatus
+
+
+class HttpResponseMock:
+    def __init__(self, status, reason, body, headers=None):
+        self.status = status
+        self.reason = reason
+        self.body = body
+
+
+class ApiExceptionsMock:
+    def create_permission_denied(self, body_message="User doesn't have access"):
```

Comment:
Is this method used somewhere?

Reply:
No now. But it will be used in other modules. For example fusion_info, where we have forbidden status handling